hi-ml-cpath/src/health_cpath/datamodules/base_module.py

```diff
@@ -318,3 +311,13 @@ def _get_dataloader(self, dataset: SlidesDataset, stage: ModelKey, shuffle: bool
             generator=generator,
             **dataloader_kwargs,
         )
+
+    def _get_tiling_on_the_fly_steps_transforms(self, stage: ModelKey) -> List[Callable]:
```

Comment:
nitpick on names - this tiling will always be "on the fly". Also, if people are meant to override it, it should not be private. How about `get_tiling_transforms`?